compiler/src/iree/compiler/Codegen/SPIRV/ConvertToSPIRVPass.cpp

```diff
@@ -430,6 +430,20 @@ void ConvertToSPIRVPass::runOnOperation() {
     }
   }
 
+  // Expand any remaining `bf16` `extf` and `trunc` patterns.
+  {
+    RewritePatternSet patterns(context);
+    arith::populateExpandBFloat16Patterns(patterns);
+    arith::BitcastOp::getCanonicalizationPatterns(patterns, context);
+    if (failed(applyPatternsAndFoldGreedily(moduleOp,
+                                            std::move(patterns)))) {
+      moduleOp.emitOpError() << "failed running expansion patterns";
```

Comment:
"failed running bf16 extf/trunc patterns" to be clear?

Reply:
Updated